extension/src/plots/webview/contract.ts

```diff
@@ -2,9 +2,10 @@ import { VisualizationSpec } from 'react-vega'
 import { Color } from '../../experiments/model/status/colors'
 
 export const PlotSizeNumber = {
-  LARGE: 500,
-  REGULAR: 400,
-  SMALL: 300
+  LARGE: 1,
+  MEDIUM: 3,
+  REGULAR: 2,
+  SMALL: 4
```

Comment:
[N] Seems strange to have medium and regular, could have large and x-large instead.

Reply:
Thanks, I was searching hard for how to call it and that's the best I could come up with.

Reply:
Went with `SMALL` and `SMALLER` instead as it wouldn't change what `REGULAR` is and it worked better for key order as well.